content/zh/docs/concepts/components.md

```diff
@@ -0,0 +1,110 @@
+---
+title: 组件
+description: 构成 OpenTelemetry 的主要组件
+aliases: [data-collection]
+weight: 20
+---
+
+OpenTelemetry 目前由以下几个主要组件构成：
+
+- [规范](#specification)
+- [Collector](#collector)
+- [特定语言的 API 和 SDK 实现](#language-specific-api--sdk-implementations)
+  - [插桩库](#instrumentation-libraries)
+  - [导出器](#exporters)
+  - [零代码插桩](#zero-code-instrumentation)
+  - [资源检测器](#resource-detectors)
+  - [跨服务传播器](#cross-service-propagators)
+  - [采样器](#sampler)
+- [K8s Operator](#k8s-operator)
+- [函数即服务（FaaS）资产](#function-as-a-service-assets)
+
+OpenTelemetry 让你无需使用特定供应商的 SDK 和工具就能生成和导出遥测数据。
+
+## 规范 {#specification}
+
+本节说明了针对所有实现的跨语言要求和期望。除了术语定义之外，规范还定义了以下内容：
+
+- **API:** 定义了用于生成和关联跟踪、指标和日志数据的数据类型和操作。
+- **SDK:** 定义了 API 特定语言实现的要求。配置、数据处理和导出概念也在这里定义。
+- **数据:** 定义了 OpenTelemetry 协议（OTLP）和与供应商无关的、遥测后端可以提供支持的语义约定。
+
+更多信息，请参见[规范](/docs/specs/)。
+
+## Collector
+
+OpenTelemetry Collector 是一个与供应商无关的代理，可以接收、处理和导出遥测数据。
+它支持以多种格式接收遥测数据（例如 OTLP、Jaeger、Prometheus 以及许多商业/专有工具）并将数据发送到一个或多个后端。
+它还支持在导出之前处理和过滤遥测数据。
+
+更多信息，请参见 [Collector](/docs/collector/)。
+
+## 特定语言的 API 和 SDK 实现 {#language-specific-api--sdk-implementations}
+
+OpenTelemetry 还提供语言 SDK，允许你使用所选语言的 OpenTelemetry API 生成遥测数据，并将这些数据导出到首选后端。
+这些 SDK 还允许你结合常见库和框架的插桩库，以便你可以将其用于应用程序中的手动插桩。
+
+更多信息，请参见[插桩操作](/docs/concepts/instrumentation/)。
+
+### 插桩库 {#instrumentation-libraries}
+
+OpenTelemetry 支持通过大量组件来为所支持的语言根据流行的库和框架生成相关遥测数据。
+例如，来自 HTTP 库的入站和出站 HTTP 请求将生成有关这些请求的数据。
+
+长期目标是流行的库能够开箱即用地进行观测，无需到拉入单独的组件中。
```

Comment:
This line seems confusing, doesn't sound like native Chinese?

Reply:
Thanks, updated